gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileAttributesTest.java

```diff
@@ -35,7 +36,6 @@
   private Path path;
   private Path dir;
 
-  /** empty test storage and make sure we use it instead of the real GCS. Create a few paths. **/
   @Before
   public void before() {
```

Comment:
Seemed easily inferred from the code itself, and not meriting a javadoc. I can put it back if you like.